app/src/main/res/layout/activity_main.xml

```diff
@@ -6,13 +6,26 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
+    <Button
+        android:id="@+id/count_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="@string/textButton_mainActivity"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
```

Comment:
instead of use         
app:layout_constraintRight_toRightOf="parent"
you should use 
app:layout_constraintEnd_toEndOf="parent"

Reply:
done